.circleci/config.yml

```diff
@@ -302,32 +302,36 @@ jobs:
             pip3 install mkdocs
             pip3 install mkdocs-material
             mkdocs build --strict
-      - when:
-          condition:
-            or:
-              - equal: [ master, << pipeline.git.branch >> ]
-              - << pipeline.git.tag >>
-          steps:
-            - add_ssh_keys
-            - run:
-                name: Generate image tag
-                command: |
-                  set -x
-                  if [ -n "$CIRCLE_TAG" ]; then
-                      DOCS_TAG="$CIRCLE_TAG"
-                  elif [ "$CIRCLE_BRANCH" == "master" ]; then
-                      DOCS_TAG="latest"
-                  else
-                      DOCS_TAG=`echo $CIRCLE_BRANCH | sed 's,/,_,g'`
-                  fi
-                  env | sort
-                  echo "export DOCS_TAG=$DOCS_TAG" >> $BASH_ENV
-            - run:
-                name: Build latest
-                command: |
-                  pip3 install mike
-                  echo $DOCS_TAG
-                  mike deploy $DOCS_TAG --remote git@github.com:esl/MongooseDocs.git --branch main --push --force
+      # - when:
+      #     condition:
+      #       or:
+      #         - equal: [ master, << pipeline.git.branch >> ]
+      #         - << pipeline.git.tag >>
+      #     steps:
+      #       - add_ssh_keys
+      #       - run:
```

Comment:
I'm still intrigued how it sneaked in here w/o earlier presence on master (it doesn't appear on the left)